### ISSUE TITLE
Code Signing xcode

### ISSUE THREAD

**Issue by sebdei** [nightscout/nightguard#182]
I'm fairly new to swift programming and hence unfortunately do not have a clue about signing apps in xcode (And I find it surprisingly complicated and frustrating).

However, I implemented the Apple Health Integration demanded in #101. It works good but before I want to hand in my Pull Request I like to test the integration on a real device. Can somebody tell me (1) if I'm able to do so and (2) how to proceed?

Thanks in advance.

**Comment by dhermanns** [comment]
Hi! Thanks for your work. You should be able to simply connect your phone with a lightning cable to your MacBook. It should be the same as running in your emulator. Real signing is needed for an Appstore-Upload only.

**Comment by sebdei** [comment]
Thanks for your quick reply. When I do this I get the following error.

<img width="896" alt="Screenshot 2022-02-04 at 22 58 32" src="https://user-images.githubusercontent.com/23342317/152608839-c861f711-d784-4e1c-90fb-c83310a0ba80.png">

I have a (non-paid) developer account set up in xcode. Sorry, I already searched half of the internet, but simply do not get how to build for a real device.

**Comment by dhermanns** [comment]
Ah ok - just change the development team to your local developer account.

**Comment by sebdei** [comment]
At some point I already tried this without success. Obviously, I tried it too often as it now tells me, I could only setup 10 App IDs every 7 days. This is ridiculous. 

Anyways, I will get back to you next week. And I'll close this until then. Thanks a lot!

**Comment by tast3rmonial** [comment]
Dear ,

We are excited to give you an update on Tastermonial.

Today we have initiated a way to evaluate food claims and for customers to get a prescription for continuous glucose monitor (CGM) sensors and experiment with different foods using our app. 

We are actively working to improve your user experiences in:

👉 Ordering a CGM

We realized the uncertainty of CGM prices in major pharmacies around the US. Therefore, our team is now working on a solution to order CGM online and ship directly to you at a competitive price.

👉 Learning from your CGM food experiment data

[See video of current app's abiity](https://youtu.be/xtaoSxdqq6U)

Here's a peek into upcoming Tastermonial app's upgrade in the next few weeks:

- the ability to run food pairing experiments wtih CGM
- automatic syncing of CGM data to view and share the blood sugar experiments
- You'll be able to use our app to scan and evaluate more foods in your pantry (we're expanding our food database)
 We'd be happy to gather your feedback that could help us in prioritizing the next launches. Click the button below to let us know your needs.

[Customer ***@***.***&firstname=&lastname=)

 

Thanks for reading!

Bude & Tastermonial Team

[Facebook](https://www.facebook.com/tastermonial)
[Instagram](https://www.instagram.com/tastermonial)

No longer want to receive these emails? [Unsubscribe](https://manage.kmail-lists.com/subscriptions/unsubscribe?a=TvyBXH&c=01FV34H3709YPRZ75K40JN9DSR&k=e80f98f8991c5852d3cd8963e83d9d2b&m=WUxwcT&r=RaxVJZe).
TASTERMONIAL
